dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -172,14 +184,15 @@ static Configuration configureFreemarker() {
           unavailable.setExceptionMessage(info.getException().getMessage());
           table.add(unavailable);
         } else {
+          Artifact artifact = entry.getKey();
           ArtifactResults results =
               generateArtifactReport(
                   configuration,
                   output,
-                  entry.getKey(),
+                  artifact,
                   entry.getValue(),
                   cache.getGlobalDependencies(),
-                  jarToLinkageReport,
+                  artifactToLinkageReports.get(artifact.toString()),
```

Comment:
toString is less than ideal as a key because it tends to change. We may want a more specific method here such as `getCoordinates`.

Reply:
Updated to `Artifacts.toCoordinates(artifact)`